teuthology/openstack/__init__.py

```diff
@@ -113,8 +113,13 @@ def get_volumes(self):
         Return the uuid of the volumes attached to the name_or_id
         OpenStack instance.
         """
-        volumes = self['os-extended-volumes:volumes_attached']
-        return [volume['id'] for volume in volumes ]
+        info = self.info or {}
+        #Some clouds use attached_volumes instead of os-extended-volumes:volumes_attached
```

Comment:
Drop this comment, the code below is pretty straightforward. It's better give this explanation  in the commit description.